src/WingetCreateCLI/Commands/UpdateLocaleCommand.cs

```diff
@@ -0,0 +1,296 @@
+
+                if (ValidateManifest(manifestDirectoryPath))
+                {
+                    if (Prompt.Confirm(Resources.ConfirmGitHubSubmitManifest_Message))
+                    {
+                        return await this.LoadGitHubClient(true) ?
+                            (commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                                originalManifests,
+                                $"Update locale: {originalManifests.VersionManifest.PackageIdentifier} version {originalManifests.VersionManifest.PackageVersion}"))
+                            : false;
+                    }
+                    else
+                    {
+                        Logger.WarnLocalized(nameof(Resources.SkippingPullRequest_Message));
+                    }
+                }
+                else
+                {
+                    return false;
+                }
+
+                return await Task.FromResult(commandEvent.IsSuccessful = true);
+            }
+            finally
+            {
+                TelemetryManager.Log.WriteEvent(commandEvent);
+            }
+        }
+
+        private static object PromptAllLocalesAsMenuSelection(DefaultLocaleManifest defaultLocale, List<LocaleManifest> localeManifests)
+        {
+            Dictionary<string, object> localeMap = new Dictionary<string, object>
+            {
+                { string.Format("{0} ({1})", defaultLocale.PackageLocale, Resources.DefaultLocale_MenuItem), defaultLocale },
+            };
+
+            foreach (var locale in localeManifests)
+            {
+                localeMap.Add(locale.PackageLocale, locale);
+            }
+
+            string selectedLocale = Prompt.Select(Resources.SelectExistingLocale_Message, localeMap.Keys.ToList());
+            return localeMap[selectedLocale];
+        }
+
+        private static void DisplayUpdatedLocales(Manifests manifest)
+        {
+            Logger.DebugLocalized(nameof(Resources.GenerateUpdatedLocalePreview_Message));
+            if (manifest.DefaultLocaleManifest != null)
+            {
+                DisplayDefaultLocaleManifest(manifest.DefaultLocaleManifest);
+            }
+
+            if (manifest.LocaleManifests != null)
+            {
+                DisplayLocaleManifests(manifest.LocaleManifests);
+            }
+        }
+
+        private Manifests PromptAndUpdateExistingLocales(Manifests originalManifests)
+        {
+            Manifests updatedLocales = new Manifests()
+            {
+                SingletonManifest = null,
+                VersionManifest = null,
+                InstallerManifest = null,
+            };
+
+            List<string> defaultPromptPropertiesForUpdateLocale = new()
+            {
+                nameof(LocaleManifest.PackageName),
+                nameof(LocaleManifest.Publisher),
+                nameof(LocaleManifest.License),
+                nameof(LocaleManifest.ShortDescription),
+            };
```

Comment:
I think having a default set of properties to prompt makes sense for a new locale, but for an update we should just show all of them. My main reason is that there will most definitely be existing locale properties that need to be updated and I'd rather not hide them. The other thing is that it may help reduce some of the complexity of prompting the user for additional properties.